common/src/config/GameConfig.cpp

```diff
@@ -103,6 +103,19 @@ bool GameConfig::detect_game_path()
         // ignore
     }
 
+    // GOG (German)
+    try {
+        RegKey reg_key(HKEY_LOCAL_MACHINE, "SOFTWARE\\GOG.com\\Games\\1207660623", KEY_READ);
```

Comment:
Nothing in this key indicates that it is a german key. 1207660623 seems to be Red Faction ID in GOG, no matter the localization. Are you sure that the old GOG key is still valid (SOFTWARE\\GOG.com\\GOGREDFACTION)?
Also this should be handled in the setup program (or in both places) because this code is never be executed unless someone installs using a zip file skipping the setup.

Reply:
For some reason, the GOG English and German setup programs each add separate keys. English adds `SOFTWARE\GOG.com\GOGREDFACTION`, and German adds `SOFTWARE\GOG.com\Games\1207660623`. The German version was released two years later in 2016, so maybe that is around the time GOG had switched over to using `Games\ProductID` over `GOG + GAMETITLE` in their setups.

The Dash setup does handle this oddity, but I saw that the launcher did not, so I added the path in for it too.

Reply:
Interesting discovery just now... Both GOG versions create `SOFTWARE\Nordic Games\Red Faction` in `HKEY_LOCAL_MACHINE`. That and the data of `INSTALL_DIR` could be used for both GOG distributions to avoid this situation.

![RFEngGOG](https://user-images.githubusercontent.com/89315313/190878076-371a02f7-5d0e-4b8f-ae58-9ae0fbfb8a77.png)

Reply:
Thanks for clarification. If setup uses those keys there is no reason to prevent the launcher from using them too.
I don't have a strong opinion whether `SOFTWARE\Nordic Games\Red Faction` should be used instead of currently used keys. It would simplify some code which is a nice thing, so if you are sure it works for both versions, you can change it. Let me know if you want this PR to be merged without additional changes.

Reply:
I'll refactor this PR to use `SOFTWARE\Nordic Games\Red Faction`. I think it is good to simplify the code where possible.